src/framework/uicomponents/qml/MuseScore/UiComponents/StyledPopupView.qml

```diff
@@ -45,6 +45,9 @@ PopupView {
         globalPos = mapToGlobal(parent.x + positionDisplacementX, parent.y + positionDisplacementY)
     }
 
+    positionDisplacementX: parent.width / 2 - width / 2
+    positionDisplacementY: parent.y + parent.height
```

Comment:
I think `parent.y` is wrong, because it's already added in onAboutToShow. Often, you won't see this because parent.y is often 0. But when it's not, this is the result: 

https://user-images.githubusercontent.com/48658420/108106112-d28ba980-708d-11eb-83e5-6172f17a87b2.mov

Reply:
fixed, thanks!